book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
```

Comment:
- [ ] Let's be consistent in our parameters. The initialize method in the Rental class expects parameter in this order, date-book-person, but here you are putting `person` before `book` (self). Kindly refactor accordingly.

  ```
  Rental.new(date, self, person)
  ```

Reply:
Thanks for this useful comment.